PROJECT

```diff
@@ -43,4 +43,20 @@ resources:
   kind: FlagSourceConfiguration
   path: github.com/open-feature/open-feature-operator/apis/core/v1alpha3
   version: v1alpha3
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: openfeature.dev
+  group: core
+  kind: FeatureFlag
+  path: github.com/open-feature/open-feature-operator/apis/core/v1beta1
+  version: v1beta1
+- api:
+    crdVersion: v1
+    namespaced: true
+  domain: openfeature.dev
+  group: core
+  kind: SidecarConfiguration
```

Comment:
I am happy with this naming

Reply:
It looks like this will change to `FeatureFlagSource` based on [this thread](https://github.com/open-feature/open-feature-operator/issues/352#issuecomment-1784552206). I think that's a better name because the source information will also be valuable to the in-process version of flagd.